ephios/api/urls.py

```diff
@@ -15,11 +15,23 @@
     ApplicationUpdate,
 )
 from ephios.api.views.events import EventViewSet
-from ephios.api.views.users import UserProfileMeView
+from ephios.api.views.users import (
+    MailBasedUserParticipationView,
+    UserParticipationView,
+    UserProfileMeView,
+)
 from ephios.extra.permissions import staff_required
 
 router = routers.DefaultRouter()
 router.register(r"events", EventViewSet)
+router.register(
+    r"users/(?P<user>[\d]+)/participations", UserParticipationView, basename="user-participations"
+)
+router.register(
+    r"users/(?P<email>[\D]+)/participations",
```

Comment:
this regex looks wrong, right? You can have digits in an email. This should really be a separate route or people would just need to search for the user ID using the email first.